packages/react/react/src/hooks/usePreservedReference.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import { renderHook } from '@testing-library/react';
+import { usePreservedReference } from './usePreservedReference';
+
+type TossObject = {
+  toss: string;
+};
+
+describe('usePreservedReference', () => {
+  const callbackReturnTrue = jest.fn(() => true);
+  const callbackReturnFalse = jest.fn(() => false);
+
+  describe('with Default Callback Function', () => {
+    it('changed to the same value', () => {
+      const tossObject: TossObject = { toss: '토스' };
+
+      const { result, rerender } = renderHook(({ value }) => usePreservedReference(value), {
+        initialProps: { value: tossObject },
+      });
+
+      rerender({ value: { toss: '토스' } });
+
+      expect(tossObject).toBe(result.current);
```

Comment:
```suggestion
      expect(result.current).toBe(tossObject);
```
- Shouldn't the target of "expect" be "result.current"?